quic/transport/ngtcp2/native/certificateverifier/custom.nim

```diff
@@ -0,0 +1,69 @@
+import ngtcp2
+import sequtils
+import ./certificateverifier
+import ../pointers
+import ../../../../helpers/openarray
+
+type
+  certificateVerifierCB* = proc(derCertificates: seq[seq[byte]]): bool {.gcsafe.}
```

Comment:
nit: `proc` -> `func`

Reply:
Compiler complained with 
```
error: func keyword is not allowed in type descriptions, use proc with {.noSideEffect.} pragma instead
```
I'll add that instead then

Reply:
https://github.com/vacp2p/nim-quic/pull/56/commits/c3e770959ed02a6d53cf978db4485f7dcce1a473